functions/zram.bats

```diff
@@ -34,6 +34,29 @@ check_zram_mounts() {
   return 0
 }
 
+@test "dev-zram" {
```

Comment:
If you want to introduce a new type of test category also define it in [CONTRIBUTING.md](https://github.com/openhab/openhabian/blob/master/CONTRIBUTING.md#test-cases). How does these test differ from the other categories?

Reply:
It's useful when you have features under development that you want to test. These dev-tests are intentionally placed at the beginning so they're first to finish and testing won't move on to the next stage unless this stage is successfull. Added this explanation to CONTRIBUTING.md.